src/themes/uswitch/theme.json

```diff
@@ -2397,7 +2403,14 @@
             "textDecoration": "underline"
           },
           "li": {
-            "borderTop": "none"
+            ":first-of-type": {
+              "marginTop": "4px"
+            },
+            "borderTop": "none",
+            "paddingTop": "0px",
```

Comment:
I believe for paddingTop and PaddingBottom you can just do paddingY: 0

Reply:
yep, just tried it and it works but that's theme-ui specific though, isn't it ? wouldn't we rather stick to standard css properties ?

Reply:
or maybe it's just me, but i find standard css more readable, in this case (and some others, like "mb") theme-ui-isms don't seem to be wroth it because they add a tiny bit of friction without having any significant benefits other than saving keystrokes

Reply:
I agree things like mb and mt are hard to read as you skim and I would avoid these however paddingY is actually helpful because it allows you to see its applying to both, this is also used across our codebase already so we would need to discuss as a team whether we want to avoid paddingY